cypress-tests/cypress/e2e/PaymentUtils/Utils.js

```diff
@@ -23,6 +24,7 @@ const connectorDetails = {
   paypal: paypalConnectorDetails,
   stripe: stripeConnectorDetails,
   trustpay: trustpayConnectorDetails,
+  datatrans:datatransConnectorDetails
```

Comment:
- Was this connector tested?
- Since this is a new connector, tests against other connectors needs to be run as well. Was that done?
- Looks like lints were not fixed either